main/main.cpp

```
@@ -9,48 +9,68 @@
+           "measured humidity: %0.2f percentRH\n",
+           temperature / 1000.0f, humidity / 1000.0f);
+}
 
 extern "C" void app_main(void)
 {
+    // Create sensor instance with default configuration (address 0x70, normal power mode)
+    SHTC3Sensor sensor = SHTC3Sensor::Builder(I2C_SCL_PIN, I2C_SDA_PIN).build();
+
     /* Initialize the i2c bus for the current platform */
-    sensirion_i2c_init();
+    if (sensor.initializeBus() == false) {
```

Clang (14.0.6)

[warning] redundant boolean literal supplied to boolean operator

(readability-simplify-boolean-expr)